public/proto/ag_pb.js

```diff
@@ -602,7 +602,7 @@ proto.User.prototype.toObject = function(opt_includeInstance) {
  * @suppress {unusedLocalVariables} f is only used for nested messages
  */
 proto.User.toObject = function(includeInstance, msg) {
-  var obj = {
+  var f, obj = {
```

Comment:
@0xf8f8ff when running `make proto` on macOS, with:
```
% protoc --version
libprotoc 3.7.1
% tsc --version
Version 3.5.2
```
This file get these minor differences.